fluss-client/src/test/java/com/alibaba/fluss/client/table/FlussTableITCase.java

```diff
@@ -207,8 +209,23 @@ void testPutAndLookup() throws Exception {
         TablePath data1PkTablePath2 =
                 TablePath.of(DATA1_TABLE_PATH_PK.getDatabaseName(), "test_pk_table_2");
         createTable(data1PkTablePath2, tableDescriptor, true);
+
         // now, check put/lookup data
-        verifyPutAndLookup(data1PkTablePath2, schema, new Object[] {"a", 1});
+        Table table2 = conn.getTable(data1PkTablePath2);
+        verifyPutAndLookup(table2, schema, new Object[] {"a", 1});
+    }
+
+    @Test
+    void testLookupForNotReadyTable() throws Exception {
+        TablePath tablePath = TablePath.of("test_db_1", "test_lookup_unready_table_t1");
+        TableDescriptor descriptor =
+                TableDescriptor.builder().schema(DATA1_SCHEMA_PK).distributedBy(10).build();
+        long tableId = createTable(tablePath, descriptor, true);
+        IndexedRow rowKey = keyRow(DATA1_SCHEMA_PK, new Object[] {1, "a"});
+        // retry until all replica ready. Otherwise, the lookup maybe fail.
+        waitAllReplicasReady(tableId, descriptor);
```

Comment:
If still wait Replicas to lookup, why this test is named "testLookupForNotReadyTable"?

Reply:
> If still wait Replicas ready to lookup, why this test is named "testLookupForNotReadyTable"?

If we consider the "bucket not ready" situation here, it may introduce instability in the tests, so some comments have been added to point the importance of adding `waitAllReplicasReady `.